### ISSUE TITLE
DL3020 Use COPY instead of ADD for URLs : False Positive

### ISSUE THREAD

**Issue by datYori** [hadolint/hadolint#693]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Should not trigger DL3020 when ADD source argument is an URL

### Actual behavior

Triggering DL3030 no matter what 

### Steps to reproduce the behavior

Write a Dockerfile with 

```
ADD https://dl.k8s.io/release/v1.21.0/bin/linux/amd64/kubectl /tmp 
```

for example

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 2.6.1-no-
```

**Additional environment details (OS, stack version, etc.)**

**Comment by m-ildefons** [comment]
Hi Yann,
I'm having trouble reproducing this. I tried the latest Docker image, the latest release binary, compiled master from source and compiled the v2.6.1 tag from source and none of those triggered DL3020 when fed your example Dockerfile. Could you please provide more context? If possible the full Dockerfile or a crafted example? What OS are you running? Are you using the docker image, the github action, release binaries or compiled from source? Do you have any configurations set?

That particular rule has received fixes wrt. edge cases in the last release, so please make sure you actually are using v2.6.1 or later.
